aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -435,7 +435,20 @@
   "handlers": {
     "create": {
       "permissions": [
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeVpcs",
+        "iam:CreateServiceLinkedRole",
+        "iam:GetRole",
+        "iam:ListRoles",
         "iam:PassRole",
+        "kms:CreateGrant",
+        "kms:DescribeKey",
+        "kms:ListAliases",
```

Comment:
do you really need these permissions for dbinstance?

Reply:
Not needed indeed. Removed. Thank you!